tsconfig.json

```diff
@@ -1,11 +1,15 @@
 {
-  "files": [
-    "src/ts/main.ts"
-  ],
   "compilerOptions": {
-    "noImplicitAny": true,
+    "outDir": "./dist",
     "target": "es5",
-    "declaration": true,
-    "outDir": "dist/js/framework"
-  }
-}
\ No newline at end of file
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "lib": ["es2015", "dom"],
+    "sourceMap": true,
+    "noImplicitAny": false,
+    "removeComments": true
```

Comment:
Why remove the comments? The minified Webpack build will not contain them anyway, and the transpiled modules can contain them for better understandability if anyone looks into them (when using modules from the NPM package).